Deprecation of Services/Table

kergomard opened the pull request:
The Technical Board proposes to advance the move to the UI-Framework. This PR defines a way to get rid of Services/Table with ILIAS 10, thus simplifying our code base and the work needed to keep a consistent appearance for our end users.

Commit:
Deprecation of Services/Table

matthiaskunkel commented:
Jour Fixe, 21 FEB 2022: We highly appreciate this suggestion to get rid of the legacy table implementation. We would prefer not to have a strict deadline for not anticipating decisions about the priority of developments for ILIAS 9 and 10 within the framework of the large developments.

kergomard committed:
Pushing Back Deadlines by one Version

Due to the delays releasing ILIAS 9 we decided
to push this back one ILIAS Version

kergomard commented:
We merge this now, as the branch of ILIAS8 has been created.
